pom.xml

```diff
@@ -367,15 +367,11 @@
             </dependency>
             <dependency>
                 <groupId>io.netty</groupId>
-                <artifactId>netty-handler</artifactId>
-                <version>${netty.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>io.netty</groupId>
-                <artifactId>netty-transport-native-epoll</artifactId>
-                <version>${netty.version}</version>
+                <artifactId>netty-bom</artifactId>
+                <version>4.1.100.Final</version>
```

Comment:
Hi @janjwerner-confluent @tunnelshade this should have been `${netty.version}` users of common pom cannot pin netty bom because of this